src/http/report.ts

```diff
@@ -1,8 +1,12 @@
-import { client } from "../database";
 import { Request, Response } from "express";
+import { reportQueue } from "../jobs/report";
 
 export const report = async (req: Request, res: Response) => {
-  const orders = await client.order.findMany({ where: {} });
+  if (!req.query?.authorization) {
+    return res.sendStatus(401);
+  }
 
-  return res.json({ orders });
+  reportQueue.add({});
+
+  return res.sendStatus(201);
```

Comment:
you can do any kind of logic here..my point was to demonstrate that you only need `reportQueue.add({});` to run your job..Inside of the add method you can pass some parameters. i'd recommend to read the Bull's docs.